src/app/client/src/app/modules/learn/components/course-consumption/course-consumption-header/course-consumption-header.component.html

```diff
@@ -155,7 +155,7 @@ <h6 class="ui small info message" *ngIf="enrolledCourse && getBatchStatus() && b
         </div>
       </div>
       <!--Activity Dashboard-->
-      <div *ngIf="isGroupAdmin" class="sb-activity-dashboard-btn-content py-8">
+      <div *ngIf="isGroupAdmin && router.url.includes('groupId')" class="sb-activity-dashboard-btn-content py-8">
```

Comment:
@abhishekpnt
please avoid using includes directly either in HTML or TS

Either use chaining for null check
Have the logic / conditions in TS and assign to a variable
Use lodash to get url from router

CC: @rajeevsathish